examples/mycoind/Makefile

```diff
@@ -1,12 +1,17 @@
-.PHONY: install protoc deps
+.PHONY: install protoc clean dist build
 
-GIT_VERSION := $(shell git describe --tags)
-BUILD_FLAGS := -ldflags "-X github.com/iov-one/weave.Version=$(GIT_VERSION)"
-TENDERMINT := ${GOBIN}/tendermint
+BUILD_VERSION ?= $(shell git describe --tags)
+BUILD_FLAGS := -ldflags "-X github.com/iov-one/weave.Version=$(BUILD_VERSION)"
 BUILDOUT ?= mycoind
-GOPATH ?= $$HOME/go
 
-TM_VERSION := v0.21.0
+
+dist: clean test build
```

Comment:
this is for consistency , right?

Reply:
mainly for the build artifact but as we are not doing any I am going to remove this.